mixes/service.js

```diff
@@ -29,4 +30,10 @@ function createService (config) {
       .then(JSON.parse)
       .then(mix => ({ ...mix, id }))
   }
+
+  // TODO: how print formatted json?
```

Comment:
to stringify "pretty" JSON (2-space indents):

```js
JSON.stringify(mix, null, 2)
```

[`JSON.stringify` docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)